ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -63,7 +63,8 @@ private class JdbcLedgerDao(
     valueSerializer: ValueSerializer,
     keyHasher: KeyHasher,
     dbType: DbType,
-    loggerFactory: NamedLoggerFactory)
+    loggerFactory: NamedLoggerFactory,
+    executionContext: ExecutionContext)
```

Comment:
We may want to name this so that it's clear this is the execution context in which Protobuf deserialization happens.

Reply:
yeah, i wasn't sure whether we should use a separate cached threadpool just for that. Right now I'm passing down the actorsystem's dispatcher, which also doesn't seem right, as it'll block scheduling of actor/stream work.